### PR TITLE
Add Assured workload folder for production environment

### PR DESCRIPTION
**Pull request by daniel-cit**
This PR  creates a folder,`fdr-production-assured-workload`, under the `fdr-production` folder  and configure  Assured Workload for this new folder

### CODE REVIEW

File: 2-environments/envs/production/assured_workload.tf [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+locals {
+  assured_workload_display_name = "${var.folder_prefix}-assured-workload"
+  enable_assured_workload       = var.assured_workload_configuration != null && var.assured_workload_configuration.enabled
```
Comment (bharathkkb) [inline 2-environments/envs/production/assured_workload.tf:19]: Cant we do?
```suggestion
  enable_assured_workload       = lookup(var.assured_workload_configuration, "enabled", false)
```

File: 2-environments/envs/production/outputs.tf [hunk 2]
```diff
@@ -38,3 +38,8 @@ output "env_secrets_project_id" {
   description = "Project for environment related secrets."
   value       = module.env.env_secrets_project_id
 }
+
+output "assured_workload_folder_display_name" {
+  description = "Assured Workload folder display name."
+  value       = local.enable_assured_workload ? local.assured_workload_display_name : ""
```
Comment (bharathkkb) [inline 2-environments/envs/production/outputs.tf:44]: `id` maybe more useful for end users

Comment (morgante) [reply 2-environments/envs/production/outputs.tf:44]: Do we need an output at all?

Comment (bharathkkb) [reply 2-environments/envs/production/outputs.tf:44]: @morgante I was thinking of usage in tests (using [gcloud assured workloads describe](https://cloud.google.com/sdk/gcloud/reference/assured/workloads/describe)) as the resource seems to have a generated UUID as name.

Comment (morgante) [reply 2-environments/envs/production/outputs.tf]: Ah that makes sense. I agree we should return ID then.

File: 2-environments/envs/production/assured_workload.tf (hunk 1 above)
Comment (daniel-cit) [reply 2-environments/envs/production/assured_workload.tf]: fixed

File: 2-environments/envs/production/outputs.tf (hunk 2 above)
Comment (daniel-cit) [reply 2-environments/envs/production/outputs.tf]: the ID of the workload added an an output + the folder id that contains the workload

**Pull request closed without merging**